ProcessLib/HeatTransportBHE/LocalAssemblers/HeatTransportBHELocalAssemblerSoil-impl.h

```diff
@@ -83,33 +84,41 @@ void HeatTransportBHELocalAssemblerSoil<ShapeFunction, IntegrationMethod>::
     auto local_K = MathLib::createZeroedMatrix<NodalMatrixType>(
         local_K_data, ShapeFunction::NPOINTS, ShapeFunction::NPOINTS);
 
-    unsigned const n_integration_points =
-        _integration_method.getNumberOfPoints();
-
     ParameterLib::SpatialPosition pos;
     pos.setElementID(_element_id);
 
+    auto const& process_data = this->_process_data;
```

Comment:
Same here *etc*.

Reply:
Actually this local process_data is not necessary. Removed.